### ISSUE TITLE
batch normalization, and input size  

### ISSUE THREAD

**Issue by hanna-kim** [fabianbormann/Tensorflow_Segmentation#30]
Hi!
I wonder why batch normalization and activation function is not used in the code.
Also, another thing that I am curious about is that filter size of 'fc_6' is 7*7 in this code. however , the filter width&height should be variable to make fc_7 1*1 vector.

Thank you.

**Comment by fabianbormann** [comment]
> I wonder why batch normalization [...] is not used in the code.

Hi @hanna-kim the answer is pretty simple: Take a look at the modification dates 😉  .. When I started the project in 2016 the batch normalization was not implemented in TensorFlow. So I didn't used it. The project was created besides of my studies, but when I started to work I simply had no time to maintain this project. Anyway... You are right, so please send me a pull request and I will accept it asap 😊
